src-os/runtime.js

```diff
@@ -0,0 +1,42 @@
+
+//Provides: ocaml_ptime_clock_period_d_ps
+function ocaml_ptime_clock_period_d_ps(_unit) {
+  return 0;
+}
+
+//Provides: ocaml_ptime_clock_current_tz_offset_s
+function ocaml_ptime_clock_current_tz_offset_s(_unit) {
+  return [0, ((new Date()).getTimezoneOffset() * -60)]
+}
+
+//Provides: ocaml_ptime_clock_now_d_ps
+//Requires: caml_int64_of_int32, caml_int64_of_float
+//Requires: caml_int64_add, caml_int64_mul
+//Requires: caml_modf_float
+//Requires: caml_raise_sys_error
+function ocaml_ptime_clock_now_d_ps(_unit) {
+  function err (ms) {
+    caml_raise_sys_error("Ptime_clock: can't represent JavaScript timestamp "+ms);
+  }
+  var dmin = 0;
```

Comment:
Why is this 0 ? In the corresponding OCaml code this is `val dmin : int = -719528`

Reply:
Typo I guess